components/landing/HighlightBox.js

```diff
@@ -1,27 +1,12 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import NLink from 'next/link'
-import { Flex, Box, Link, theme } from 'ooni-components'
-import { Text } from 'rebass'
+import { Flex, Box, Link, theme, Text } from 'ooni-components'
```

Comment:
I thought we wanted to go in the direction of using the components directly imported from `rebass` in the future. Is this no longer the case?

Reply:
We will be doing that going forward, after this branch lands to avoid bloating this PR further. This particular change is a reversal of a temporary fix from earlier when `<Text/>` from `ooni-components` did not allow changing the `font-weight` style which it does now. It just make the usage uniform across the project.